app/models/report.rb

```diff
@@ -25,4 +25,6 @@ class Report < ApplicationRecord
 
   validates :user_id, presence: true
   validates :period_id, presence: true
```

Comment:
Unrelated to this PR, but if you have `belongs_to` defined, [a presence validation is added by default](https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/belongs_to).